src/reset.css

```diff
@@ -0,0 +1,11 @@
+body {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
you can combine selectors using coma
```suggestion
body, h2, p {
  margin: 0;
}
```